### PR TITLE
Version 4.2.0

### PR DESCRIPTION
**Pull request by indiesquidge**

### CODE REVIEW

**Comment by indiesquidge** [comment]
@lencioni @noratarano I ran the `version:minor` script but couldn't run directly on `master` like [other version updates][1] have done since I don't have write access to that protected branch.

Would y'all mind taking a look and helping me publish this minor version? Thanks

[1]: https://github.com/airbnb/react-with-styles/commit/59a9548c7d18d8bf8bcf8a99cea85548114edf73

**Comment by ljharb** [comment]
Once this PR is approved, you can fast-forward merge *on the command line* and `git push`, and it will merge the PR, and then you can tag that commit and push the tag.

**Review by ljharb — APPROVED** [review]

**Comment by lencioni** [comment]
Published as https://www.npmjs.com/package/react-with-styles/v/4.2.0

**Pull request merged**
